### ISSUE TITLE
`.chosen-results` dropdown menus not properly receiving focus in all necessary components

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2335]
`ul` within dropdowns that have the class `.chosen-results` are scrollable regions that are not properly receiving keyboard focus. They have no focus indicators, collapsed or expanded, when navigating via keyboard. By guessing where focus is, it is possible to open, scroll, and make a selection via keyboard, but that functionality is not always behaving normally. At times, as originally observed in [Testing: Item stubs
#2323 (comment)](https://github.com/omeka/omeka-s/issues/2323#issuecomment-3084655745), using space/enter to navigate these menus causes unexpected events, such as form submissions or lost focus. Screen readers map focus indicators onto the input that might better illuminate the issue:

When focus is brought to the dropdown, it does not actually grab onto the dropdown menu. Instead, it's directed to the search bar within that element:
<img width="553" height="118" alt="Closed dropdown menu. A black focus outline sits below the menu on an invisible element" src="https://github.com/user-attachments/assets/90059df0-7f22-45b0-876c-9d7c9871cb3e" />

This is better seen by expanding the menu. While in the element, focus will not move outside of the search bar to any other components of the menu (except to tab to the next element):
<img width="560" height="338" alt="Open dropdown menu. A black focus outline is around the search bar in the menu" src="https://github.com/user-attachments/assets/a3401cb4-70f8-46d3-964a-9f9cf8b1ad3e" />
Thus, screen reader users cannot meaningfully access the information the dropdown provides. It is only really possible to operate the dropdown by completing the search with a known choice, so again, I guess the issue here really lies in the lack of focusability of the `ul` in the dropdown, perhaps the `select` as a whole.

**Comment by aviannamiller** [comment]
In the interim, the updated focus outline looks to be 1px in width. Can we bump it up to at least 2px?

**Comment by kimisgold** [comment]
Chosen 3.0 was just released, so I'm going to look into the updates for any accessibility improvements. The developer also responded to feedback on another issue regarding aria labeling, so I'm hopeful we're still able to progress without switching plugins.
